README.md

```diff
@@ -15,12 +15,6 @@ Add this line to your application's Gemfile:
 gem 'rollbar', '~> 2.7.0'
 ```
 
-If you are not using JRuby we suggest using [Oj](https://github.com/ohler55/oj) for JSON serialization. In order to install Oj you can add this line to your Gemfile:
```

Comment:
@ootoovak sorry, I don't understand the purpose of this PR. In this sentence we explain to use Oj only if you are *not* using JRuby.